turbopack/crates/turbopack-ecmascript/src/references/mod.rs

```diff
@@ -2402,51 +2402,13 @@ async fn value_visitor_inner(
             "Buffer" => JsValue::WellKnownObject(WellKnownObjectKind::NodeBuffer),
             _ => return Ok((v, false)),
         },
-        JsValue::Module(ModuleValue {
-            module: ref name, ..
-        }) => {
-            if *compile_time_info.environment().node_externals().await? {
-                // TODO check externals
```

Comment:
You removed that TODO but I don't see any change in logic?

Reply:
Ah nice catch, restored the comment